index.bs

```diff
@@ -474,6 +462,59 @@ To perform a <dfn abstract-op>cross-origin resource policy check</dfn> given a [
         extensions, and I think it'll be more difficult to ship them after inverting the
         error-handling behavior.
 
+To perform a <dfn abstract-op>cross-origin resource policy check</dfn> given a [=request=]
+(|request|) and a [=response=] (|response|), run these steps:
+
+1.  Let |embedder policy| be |request|'s [=request/client=]'s
+    [=environment settings object/embedder policy=].
+
+2.  If |request|'s [=request/reserved client=] is not `null`, then set |embedder policy|
+    to a new [=/embedder policy=].
+
+3.  If |embedder policy|'s [=embedder policy/report only reporting endpoint=] is not `null` and the
+    result of running [$cross-origin resource policy internal check] with
+    [=embedder policy/report only value=], |request| and |response| is `blocked`, then run these
+    steps:
+
+    1. Let |blocked url| be the first URL of |request|'s [=request/URL list=].
+
+    2. Set |blocked url|'s [=url/username=] to the empty string, and its [=url/password=] to `null`.
+
+    3. Set |serialized blocked url| be the result of executing the
+       [URL serializer](https://url.spec.whatwg.org/#concept-url-serializer) on |blocked url| with
+       the |exclude fragment flag| set.
+
+    4. Let |body| be a new object containing the following properties with keys:
+
+        * key: "`blocked`", value: |serialized blocked url|.
+
+    5. [Queue](https://w3c.github.io/reporting/#queue-report) |body| as "`coep-corp`" for
+       |embedder policy|'s [=embedder policy/report only reporting endpoint=] on |request|'s
+       [=request/client=].
+
+4.  Let |result| be the result of running [$cross-origin resource policy internal check] with
+    [=embedder policy/value=], |request| and |response| is `blocked`.
```

Comment:
This sentence doesn't make sense as written. Is "is `blocked`" from an earlier edit?

Reply:
Yes, thank you. Fixed.